### DISCUSSION TITLE
user is too new???

### DISCUSSION THREAD

**Discussion by ghost** [cirruslabs/cirrus-ci-docs#1096] [General]
i wanna tried PR orangefox recovery but always aborted and he said user is too new?? how to solve this

**Comment by fkorotkov** [comment]
It's a 48 hours incubating period because periodically we see crypto miners creating batches of new users on GitHub and trying to mine coins using Cirrus CI and other CIs that provide free compute for OSS.

But you should be good to go with creating a PR and/or using [connected compute services](https://cirrus-ci.org/guide/supported-computing-services/), the incubation only affects your personal repositories.
